app/pages/collections/index.cjsx

```diff
@@ -1,54 +1,57 @@
 counterpart = require 'counterpart'
 React = require 'react'
+TitleMixin = require '../../lib/title-mixin'
 Translate = require 'react-translate-component'
-{Link, IndexLink} = require 'react-router'
+CollectionsNav = require './nav'
+classNames = require 'classnames'
 
 counterpart.registerTranslations 'en',
+  projectcollectionsPage:
+    title: '%(user)s %(project)s Collections'
+    countMessage: 'Showing %(count)s favorites collections.'
+    button: 'View Favorites Collection'
```

Comment:
because why? You don't think we should distinguish between an ordinary collection and a favorites collection? I thought it could be useful to distinguish.....